tests/tracking/test_fluent.py

```diff
@@ -397,3 +398,61 @@ def test_start_run_existing_run_deleted(empty_active_run_stack):
     with mock.patch.object(MlflowClient, "get_run", return_value=mock_run):
         with pytest.raises(MlflowException):
             start_run(run_id)
+
+
+@mock.patch("time.time", lambda: -1)
+@pytest.mark.parametrize("fluent_fn,fluent_kwargs,expected_store_kwargs", [
+    (mlflow.log_metrics, {"metrics": {"a": 0.1}},
+     {"run_id": "my-uuid", "metrics": [Metric("a", 0.1, -1)], "params": [], "tags": []}),
+    (mlflow.log_params, {"params": {"b": "c"}},
+     {"run_id": "my-uuid", "metrics": [], "params": [Param("b", "c")], "tags": []}),
+    (mlflow.set_tags, {"tags": {"d": "e"}},
+     {"run_id": "my-uuid", "metrics": [], "params": [], "tags": [RunTag("d", "e")]}),
+])
+def test_log_batch_apis_delegate_to_store(
+        tracking_uri_mock,  # pylint: disable=unused-argument
+        fluent_fn, fluent_kwargs, expected_store_kwargs):
+    # Test that the log_metrics, log_params, set_tags fluent APIs call into to an AbstractStore
+    # instance's log_batch implementation
+    mock_run = mock.Mock()
+    mock_run.info.lifecycle_stage = LifecycleStage.ACTIVE
+    mock_run.info.run_uuid = "my-uuid"
+    mock_store = mock.Mock()
+    with mock.patch.object(mlflow.tracking.fluent, "_get_or_start_run", return_value=mock_run), \
+            mock.patch.object(mlflow.tracking.utils, "_get_store", return_value=mock_store):
+        fluent_fn(**fluent_kwargs)
+        mlflow.tracking.fluent._get_or_start_run.assert_called_once()
+        # Assert backing store log_batch was called with expected arguments
+        _, store_call_kwargs, = mock_store.log_batch.call_args
+        assert store_call_kwargs["run_id"] == expected_store_kwargs["run_id"]
+        assert [dict(m) for m in store_call_kwargs["metrics"]] == \
+               [dict(m) for m in expected_store_kwargs["metrics"]]
+        assert [dict(p) for p in store_call_kwargs["params"]] == \
+               [dict(p) for p in expected_store_kwargs["params"]]
+        assert [dict(t) for t in store_call_kwargs["tags"]] == \
+               [dict(t) for t in expected_store_kwargs["tags"]]
+
+
+def test_log_batch_client_apis_delegate_to_store(
```

Comment:
Same question about the necessity of including this test. This also doesn't seem to test fluent API behavior, so I wonder if it belongs in the fluent API test suite.